apps/ios/GuideDogs/Code/Behaviors/Helpers/CalloutStateMachine.swift

```diff
@@ -337,32 +338,38 @@ extension CalloutStateMachine {
                 sounds = callout.sounds(for: strongSelf.geo?.location, automotive: strongSelf.motionActivityContext.isInVehicle)
             }
             
+            //sounds is called here
+            
+            let previousSound: Sounds = sounds
+            
             strongSelf.audioEngine.play(sounds) { (success) in
                 calloutGroup.delegate?.calloutFinished(callout, completed: success)
-                
+
                 guard strongSelf.currentState != State.stopping.rawValue else {
                     GDLogVerbose(.stateMachine, "Callout interrupted. Stopping...")
                     strongSelf.stateMachine.fireEvent(.stopped)
                     calloutGroup.onComplete?(false)
                     return
                 }
-                
+
                 guard strongSelf.currentState != State.off.rawValue else {
                     GDLogVerbose(.stateMachine, "Callouts immediately interrupted. Cleaning up...")
                     calloutGroup.onComplete?(false)
                     return
                 }
-                
+
                 guard success else {
                     GDLogVerbose(.stateMachine, "Callout did not finish playing successfully. Terminating state machine...")
                     calloutGroup.onComplete?(false)
                     strongSelf.stateMachine.fireEvent(.failed)
                     return
                 }
                 
-                strongSelf.stateMachine.fireEvent(.delayCalloutAnnounced)
```

Comment:
Except maybe this deletion is meaningful?

Reply:
(Based on the comment below) I don't know how this ended up getting deleted. I experimented a lot with the code when I was just getting started, so maybe I deleted this in that process. I have re-added it, so you should see the line added back on the next push of this branch.